ckanext/lacounts/templates/home/snippets/community.html

```diff
@@ -1,8 +1,15 @@
+{# TODO: It doesn't work inside `macros/editable.html' #}
```

Comment:
@amercader 
I get brutal `internal error` on any attempt of moving this duplicated code to a `macros` or `snippets` file.